portfolio/src/main/webapp/script.js

```diff
@@ -75,9 +75,37 @@ function addRandomFact(){
     if(facts.length == 0){
         factContainer.innerText = "Thats all the facts about me !";
     } else {
-    // Pop element via index. 
-    facts.splice(idx,1);
-    // Add it to the page.
-    factContainer.innerText = fact;
+        // Pop element via index. 
+        facts.splice(idx,1);
+        // Add it to the page.
+        factContainer.innerText = fact;
     }
 }
+
+function getMessages(){
+    fetch('/data').then(response => response.json()).then((comment) => {
+    const dataListElement = document.getElementById('comment-container');
+    dataListElement.innerHTML = '';
+    comments.forEach((comment) => {
+      dataListElement.appendChild(createTableRowElement(comment));
+    })
+  });
+}
+
+/** 
+ * Creates an table row element,<tr><td> </td></tr>, containing text. 
+ */
+function createTableRowElement(commentObj) {
```

Comment:
Regarding the name, is it important that this method return a tr element or is that an implementation detail? To me this could be an opaque method that takes a comment and returns the HTML used to display it.

Reply:
Acknowledged

Reply:
How about `createElementForComment`? The fact that it's a table is an implementation detail of the method.